keps/sig-network/0752-endpointslices/README.md

```diff
@@ -209,21 +211,14 @@ type Endpoint struct {
     // endpoint.
     // +optional
     TargetRef *v1.ObjectReference `json:"targetRef,omitempty" protobuf:"bytes,4,opt,name=targetRef"`
-    // topology contains arbitrary topology information associated with the
-    // endpoint. These key/value pairs must conform with the label format.
-    // https://kubernetes.io/docs/concepts/overview/working-with-objects/labels
-    // Topology may include a maximum of 16 key/value pairs. For endpoints
-    // backed by Kubernetes Pods, This may include, but is not limited to the
-    // following well known keys:
-    // * kubernetes.io/hostname: the value indicates the hostname of the node
-    //   where the endpoint is located. This should match the corresponding
-    //   node label.
-    // * topology.kubernetes.io/zone: the value indicates the zone where the
-    //   endpoint is located. This should match the corresponding node label.
-    // * topology.kubernetes.io/region: the value indicates the region where the
-    //   endpoint is located. This should match the corresponding node label.
+    // nodeName represents the name of the Node hosting this endpoint. This can
+    // be used to determine endpoints local to a Node. This field can be enabled
+	  // with the EndpointSliceNodeName feature gate.
+	  // +optional
+	  NodeName *string `json:"nodeName,omitempty" protobuf:"bytes,6,opt,name=nodeName"`
+    // zone of the name of the Zone this endpoint exists in.
     // +optional
-    Topology map[string]string `json:"topology,omitempty" protobuf:"bytes,5,opt,name=topology"`
```

Comment:
We have special tags for it, e.g.:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go#L4573

In this case, it will be something like:

// +k8s:deprecated=topology,protobuf=5

Reply:
Please fix

Reply:
Added a deprecated comment on line 218 with a short description of what the field used to be.